ABUNavigator:
  ros__parameters:
    i2c_bus_number: 1
    TOF_front_EN:  19
    TOF_left_EN:   13
    TOF_right_EN:  26

    twist_output_topic: "/cmd_vel_nav"
    
    motion_A1A2_period: 75
    motion_A1A2_velocity: 0.68

    motion_A2A3_period: 59
    motion_A2A3_velocity: 0.6

    motion_A2A2_max_velocity: 0.45
    motion_A2A2_min_velocity: 0.25
    motion_A2A2_Kp: 0.0008
    
    motion_A2RED_normal_velocity: 0.55
    motion_A2RED_slow_velocity: 0.3
    motion_A2RED_Kt: 17

    motion_A2BLU_normal_velocity: 0.55
    motion_A2BLU_slow_velocity: 0.3
    motion_A2BLU_Kt: 17
